flake.nix

```diff
@@ -94,6 +117,40 @@
           '';
         });
 
+      apps = perSystem (system: rec {
+        default = ogmios-datum-cache;
+        ogmios-datum-cache = {
+          type = "app";
+          program = "${
+              self.packages.${system}.${hsPackageName}
+            }/bin/ogmios-datum-cache";
+        };
+        vm = {
+          type = "app";
+          program =
+            "${self.nixosConfigurations.test.config.system.build.vm}/bin/run-nixos-vm";
+        };
+      });
+
+      nixosModules.ogmios-datum-cache = { pkgs, lib, ... }: {
+        imports = [ ./nix/ogmios-datum-cache-nixos-module.nix ];
+        nixpkgs.overlays = [
+          (_: _: {
+            ogmios-datum-cache = self.packages.${pkgs.system}.${hsPackageName};
+          })
+        ];
+      };
+
+      nixosConfigurations.test = nixpkgs.lib.nixosSystem {
+        system = "x86_64-linux";
+        modules = [
+          inputs.cardano-node.nixosModules.cardano-node
+          inputs.ogmios.nixosModules.ogmios
+          inputs.self.nixosModules.ogmios-datum-cache
```

Comment:
We could simply use `self.nixosModules.ogmios-datum-cache`.

Reply:
fixed